hhs_oauth_server/settings/base.py

```diff
@@ -654,7 +654,7 @@
 
 # Change these for production
 USER_ID_SALT = env("DJANGO_USER_ID_SALT", "6E6F747468657265616C706570706572")
-USER_ID_ITERATIONS = int(env("DJANGO_USER_ID_ITERATIONS", "2"))
+USER_ID_ITERATIONS = 2
```

Comment:
@ajshred This needs changed back or the deployment will fail.

I've also been needing to hard code this value to run locally after some recent change. Just looking for how to solve it with a change in the repository.  Are you OK with me pushing up that change as part of your PR or should this be something separate?